packages/web/src/hooks/useSyntaxHighlightingExtension.ts

```diff
@@ -60,11 +104,109 @@ export const getSyntaxHighlightingExtension = (language: string) => {
+        case "jupyter notebook":
+            return json();
+        case "less":
+            return less();
+        case "liquid":
+            return liquid();
         case "markdown":
             return markdown();
+        case "php":
+            return php();
+        case "python":
+            return python();
+        case "rust":
+            return rust();
+        case "sass":
+            return sass();
+        case "sql":
+            return sql();
+        case "vue":
+            return vue();
+        case "xml":
+            return xml();
+        case "yaml":
+        case "oasv2-yaml":
+        case "oasv3-yaml":
+            return yaml();
+
+        // Legacy CodeMirror 5 modes
+        case "clojure":
+            return StreamLanguage.define(clojure);
+        case "cmake":
+            return StreamLanguage.define(cmake);
+        case "cobol":
+            return StreamLanguage.define(cobol);
+        case "coffeescript":
+            return StreamLanguage.define(coffeeScript);
+        case "common lisp":
+            return StreamLanguage.define(commonLisp);
+        case "d":
+            return StreamLanguage.define(d);
+        case "dockerfile":
+            return StreamLanguage.define(dockerFile);
+        case "diff":
+            return StreamLanguage.define(diff);
+        case "elm":
+            return StreamLanguage.define(elm);
+        case "erlang":
+            return StreamLanguage.define(erlang);
+        case "fortran":
+            return StreamLanguage.define(fortran);
+        case "groovy":
+            return StreamLanguage.define(groovy);
+        case "haskell":
+            return StreamLanguage.define(haskell);
+        case "jinja":
+            return StreamLanguage.define(jinja2);
+        case "julia":
+            return StreamLanguage.define(julia);
+        case "livescript":
+            return StreamLanguage.define(liveScript);
+        case "lua":
+            return StreamLanguage.define(lua);
+        case "nginx":
+            return StreamLanguage.define(nginx);
+        case "pascal":
+            return StreamLanguage.define(pascal);
+        case "perl":
+            return StreamLanguage.define(perl);
+        case "powershell":
+            return StreamLanguage.define(powerShell);
+        case "protocol buffer":
+            return StreamLanguage.define(protobuf);
+        case "pug":
+            return StreamLanguage.define(pug);
+        case "puppet":
+            return StreamLanguage.define(puppet);
+        case "r":
+            return StreamLanguage.define(r);
+        case "rpm spec":
+            return StreamLanguage.define(rpmSpec);
+        case "ruby":
+            return StreamLanguage.define(ruby);
+        case "scheme":
+            return StreamLanguage.define(scheme);
+        case "shell":
+            return StreamLanguage.define(shell);
+        case "swift":
+            return StreamLanguage.define(swift);
+        case "tcl":
```

Comment:
RE @NasaGeek @er0k : tcl syntax-highlighting support incoming thanks to @KonradStanski 😃